.travis.yml

```diff
@@ -30,7 +31,14 @@ cache:
   - "$HOME/.composer/cache"
 
 before_install:
-- phpenv config-rm xdebug.ini
+- >
+  if [[ $DEPENDENCIES = "latest" && ${TRAVIS_PHP_VERSION:0:3} = "7.2" ]]; then
+    export PHPUNIT_OPTIONS="--coverage-clover=/tmp/coverage.clover";
+    export COVERAGE=yes
```

Comment:
Please let's model this via the build matrix instead. That's what the build matrix is for, and that's where this should be located to minimize the amount of surprise. :-)

Reply:
@oliverklee  with configuration like this:
```yaml
  matrix:
  - DEPENDENCIES=latest
  - DEPENDENCIES=oldest
```
I don't know how to define configuration where only 1 build will make a code coverage. For example if i change a configuration like this:
```yaml
  matrix:
  - DEPENDENCIES=latest COVERAGE=yes
  - DEPENDENCIES=oldest
```
Travis will make 3 build with COVERAGE=yes environment.

The only option I know is to change configuration to be:
```yaml
matrix:
    include:
        - php: 7.0
          env: DEPENDENCIES=latest COVERAGE=yes
        - php: 7.0
          env: DEPENDENCIES=oldest
        - php: 7.1
          env: DEPENDENCIES=latest
        - php: 7.1
          env: DEPENDENCIES=oldest
        - php: 7.2
          env: DEPENDENCIES=latest
        - php: 7.2
          env: DEPENDENCIES=oldest
```
Any suggestion?

Reply:
Either that, or you can try using matrix: exclude.